source/_components/device_tracker.bluetooth_le_tracker.markdown

```diff
@@ -32,4 +32,12 @@ Some BTLE devices (e.g. fitness trackers) are only visible to the devices that t
 
 BTLE tracking requires root privileges.
 
+For running Home Assistant as non root user we can give python3 the missing capabilities to access the bluetooth stack. Quite like setting the setuid bit (see [Stack Exchange](http://unix.stackexchange.com/questions/96106/bluetooth-le-scan-as-non-root) for more information).
+```bash
```

Comment:
Please wrap your code blocks with a new line before and after or else our rendered will not pick it up.